<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>book</class>
 <widget class="QDialog" name="book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>472</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>891</width>
    <height>472</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>891</width>
    <height>472</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查询修改更新书目</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>书号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_id">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>书名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_author">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_press">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>藏书量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加书目</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="book_all">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>书号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deleteid">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>651</width>
     <height>341</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>书号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>作者</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出版社</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>总存量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>剩余量</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="freshen">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextpage">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lastpage">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>430</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>共</string>
   </property>
  </widget>
  <widget class="QLabel" name="now">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>430</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="total">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>430</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="end">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>尾页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="begin">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>首页</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>31</width>
     <height>421</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="kind">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>50</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>book's id</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>author</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>book concern</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="seekline">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>49</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="seek">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;添加和更改书目信息：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;删除书籍信息：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;所有书籍信息：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/mainwindow/book.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="bexit">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/mainwindow/20080528160815502.gif);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/mainwindow/lms.ico);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>label</zorder>
  <zorder>book_id</zorder>
  <zorder>label_2</zorder>
  <zorder>book_name</zorder>
  <zorder>book_author</zorder>
  <zorder>label_3</zorder>
  <zorder>book_press</zorder>
  <zorder>label_5</zorder>
  <zorder>add</zorder>
  <zorder>label_6</zorder>
  <zorder>book_all</zorder>
  <zorder>label_7</zorder>
  <zorder>deleteid</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>table</zorder>
  <zorder>delete_2</zorder>
  <zorder>freshen</zorder>
  <zorder>nextpage</zorder>
  <zorder>lastpage</zorder>
  <zorder>label_4</zorder>
  <zorder>now</zorder>
  <zorder>label_9</zorder>
  <zorder>total</zorder>
  <zorder>label_11</zorder>
  <zorder>end</zorder>
  <zorder>begin</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>kind</zorder>
  <zorder>seekline</zorder>
  <zorder>seek</zorder>
  <zorder>label_8</zorder>
  <zorder>label_10</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>time</zorder>
 </widget>
 <tabstops>
  <tabstop>book_id</tabstop>
  <tabstop>book_name</tabstop>
  <tabstop>book_author</tabstop>
  <tabstop>book_press</tabstop>
  <tabstop>book_all</tabstop>
  <tabstop>deleteid</tabstop>
  <tabstop>add</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>table</tabstop>
  <tabstop>delete_2</tabstop>
  <tabstop>freshen</tabstop>
  <tabstop>bexit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
